doc/data-files-description.md

```diff
@@ -15,83 +15,84 @@ This document contains information about all data files associated with this pro
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file described in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py) |
+|`whole_exome_agilent_designed_120.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_designed_120.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh) |
+|`whole_exome_agilent_designed_120.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file described in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py) |
+|`whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_plus_tcga_6k.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh) |
+|`whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/). Script to retreive file described in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/get-tcga-capture_kit.py) |
```

Comment:
It seems to me that the `Origin` column for all of these should be `tcga-capture-kit-investigation`?

Reply:
> We can put something like GDC, since we got via GDC?

This would make sense I think.

> The padded files were created based on the original files..

Is there not a script that records this?

Reply:
Ok, I think I addressed everything, except the padding scripts, which @migbro will add to the OpenPBTA-workflows repository since they are currently _not_ captured within the SNV workflow as I thought.